scripts/exportFromSketch.js

```diff
@@ -38,39 +38,11 @@ const diffToArray = stdout => stdout
 	);
 
 //
-// Because we `diff` against `master` to select which files to export,
-// bail out if `src/sketch/` is in dirty state.
+// Export artboards from all sketch files
+// based on dist config from `exportConfig.json`
 //
 exec(
-	`git status --porcelain ${SRC_DIR}`,
-	(error, result) => {
-			if (error !== null) throw new Error(`exec error: ${error}`);
-
-			const localChanges = result
-				.split('\n')
-				.filter(f => f);
-
-			if (localChanges.length) {
-				console.error('\n---------------------------------------------------------------');
-				console.error(`${SRC_DIR} is in a dirty state.`);
-				console.error('You must commit sketch files before exports can build.');
-				console.error(`Please commit your changes in ${SRC_DIR} and try again.`);
-				console.error('---------------------------------------------------------------\n');
-				process.exit(1);
-			}
-	}
-);
-
-//
-// only build files that have changed
-//
-// `--diff-filter=ACMRT`
-// A - added
-// C - copied
-// M - modified
-// T - type (mode) change
-exec(
-	`git diff master --diff-filter=ACMT --name-only ${SRC_DIR}`,
+	`ls ${SRC_DIR}`,
```

Comment:
This is the big change. We now just grab all sketch file names from the `src` dir instead of reading a diff against master.